mockld/sdk_data.go

```diff
@@ -63,6 +71,23 @@ func (b blockingUnavailableSDKData) Serialize() []byte { return nil }
 // We use this for both regular server-side SDKs and the PHP SDK.
 type ServerSDKData map[DataItemKind]map[string]json.RawMessage
 
+func (s ServerSDKData) AsFDv2SDKData(t *ldtest.T) FDv2SDKData {
```

Comment:
The data model in the parameterized tests used to match the SDK format. That doesn't work anymore. For now, I do a simple translation. Ideally we would do this when loading the files.